Framework/MainDriverApi.py

```diff
@@ -795,12 +795,11 @@ def zip_and_delete_tc_folder(
 
 def cleanup_driver_instances():  # cleans up driver(selenium, appium) instances
     try:  # if error happens. we don't care, main driver should not stop, pass in exception
-
-        if CommonUtil.teardown and shared.Test_Shared_Variables("selenium_driver"):
-            import Framework.Built_In_Automation.Web.Selenium.BuiltInFunctions as Selenium
-            driver = shared.Remove_From_Shared_Variables("selenium_driver")
-            if driver not in failed_tag_list:
-                Selenium.Tear_Down_Selenium()
+        import Framework.Built_In_Automation.Web.Selenium.BuiltInFunctions as Selenium
+        try:
+            Selenium.Tear_Down_Selenium()
```

Comment:
| Scenario                                                       | Action | Test Result |
| -------------------------------------------------------------- | ------ | ----------- |
| Debug teardown - Normal                                        | Old    | Pass        |
| Debug teardown if previous session was forcefully closed       | Old    | Pass        |
| Deploy teardown if previous debug session was forefully closed | Old    | Pass        |
| Deploy teardown after running testcase in deploy mode          | Old    | Pass        |
| Debug continuity if Teardown is False                          | Old    | Pass        |
| In deploy initial Teardown is default                          | Old    | Pass        |
| Debug teardown - Normal                                        | New    | Pass        |
| Debug teardown if previous session was forcefully closed       | New    | Pass        |
| Deploy teardown if previous debug session was forefully closed | New    | Pass        |
| Deploy teardown after running testcase in deploy mode          | New    | Pass        |
| Debug continuity if Teardown is False                          | New    | Pass        |
| In deploy initial Teardown is default                          | New    | Pass        |

Reply:
Can you modify the cells to say `v1` and `v2` instead of old and new? that will make it easier to understand.